src/css/profile/mobile/common/core.less

```diff
@@ -148,12 +148,19 @@
 
 	.ui-content-area {
 		background-color: var(--background-area-color);
-		mask-border: url(images/0_Round_corner/round.svg) 77 round;
-		mask-border-width: 26 * @px_base;
```

Comment:
I removed mask image because the wrong line appeared in specific resolution.
![image](https://user-images.githubusercontent.com/29908741/76597189-32ef9e00-6543-11ea-8759-0f52c7b2cb3e.png)
Is there any problem to remove the mask border ? I think it can be covered by radius border.. :(

Reply:
You have to check svg files for corners eg.  `images/0_Round_corner/bottom_left.svg`. If you sure that the rounding effect will be the same you can use `border-radius`. At the moment the effect seems the same.

Reply:
I removed 'round' from the mask-border and fixed issue.